src/core/ioc.py

```diff
@@ -1,43 +1,59 @@
+
 from src.api.controllers.chef_controller import ChefController
 from src.api.controllers.ingredient_controller import IngredientController
 from src.api.controllers.inventory_controller import InventoryController
-from src.api.controllers.inventory_ingredient_controller import \
-    InventoryIngredientController
+from src.api.controllers.inventory_ingredient_controller import (
+    InventoryIngredientController,
+)
 from src.api.controllers.order_controller import OrderController
 from src.api.controllers.order_detail_controller import OrderDetailController
-from src.api.controllers.order_status_history_controller import \
-    OrderStatusHistoryController
+from src.api.controllers.order_status_history_controller import (
+    OrderStatusHistoryController,
+)
 from src.api.controllers.product_controller import ProductController
-from src.api.controllers.product_ingredient_controller import \
-    ProductIngredientController
+from src.api.controllers.product_ingredient_controller import (
+    ProductIngredientController,
+)
 from src.core.order_manager import OrderManager
 from src.lib.repositories.impl.chef_repository_impl import ChefRepositoryImpl
-from src.lib.repositories.impl.ingredient_repository_impl import \
-    IngredientRepositoryImpl
-from src.lib.repositories.impl.inventory_ingredient_repository_impl import \
-    InventoryIngredientRepositoryImpl
-from src.lib.repositories.impl.inventory_repository_impl import \
-    InventoryRepositoryImpl
-from src.lib.repositories.impl.order_detail_repository_impl import \
-    OrderDetailRepositoryImpl
+from src.lib.repositories.impl.ingredient_repository_impl import (
+    IngredientRepositoryImpl,
+)
+from src.lib.repositories.impl.inventory_ingredient_repository_impl import (
+    InventoryIngredientRepositoryImpl,
+)
+from src.lib.repositories.impl.inventory_repository_impl import InventoryRepositoryImpl
+from src.lib.repositories.impl.order_detail_repository_impl import (
+    OrderDetailRepositoryImpl,
+)
 from src.lib.repositories.impl.order_repository_impl import OrderRepositoryImpl
-from src.lib.repositories.impl.order_status_history_repository_impl import \
-    OrderStatusHistoryRepositoryImpl
-from src.lib.repositories.impl.product_ingredient_repository_impl import \
-    ProductIngredientRepositoryImpl
-from src.lib.repositories.impl.product_repository_impl import \
-    ProductRepositoryImpl
+from src.lib.repositories.impl.order_status_history_repository_impl import (
+    OrderStatusHistoryRepositoryImpl,
+)
+from src.lib.repositories.impl.product_ingredient_repository_impl import (
+    ProductIngredientRepositoryImpl,
+)
+from src.lib.repositories.impl.product_repository_impl import ProductRepositoryImpl
 
 
 class Ioc:
     def __init__(self):
+
         self._instance_ioc = {}
+        self._init_orm_session()
         self._init_order_manager()
         self._init_repositories()
         self._init_controllers()
 
+    def _init_orm_session(self):
```

Comment:
This configuration should not be here. Could create a file like `sqlalchemy_cofig` and then do any related configurations within that file?

Reply:
done